src/services/swap_service.ts

```diff
@@ -99,7 +99,12 @@ export class SwapService {
             makerAssetAmount,
             totalTakerAssetAmount,
             protocolFeeInWeiAmount: protocolFee,
+            gas,
```

Comment:
Maybe should pull this from worst case, in case we decide to make them different.